src/modules/backend/task.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable import/no-mutable-exports */
+
+let tasks = [];
+
+const loadTasks = () => {
+  let load = JSON.parse(localStorage.getItem('tasks'));
+
+  if (load == null) {
+    load = [];
+  }
+
+  tasks = load;
+
+  return tasks;
+};
+
+const clearTasks = () => {
+  tasks = [];
+};
+
+const addTask = (description, completed, index) => {
+  tasks.push({ description, completed, index: parseInt(index, 10) });
+};
+
+const store = () => {
+  const json = JSON.stringify(tasks);
+  localStorage.setItem('tasks', json);
+};
+
+const updateTaskCompleted = (index, check) => {
+  const taskInTasks = tasks.find((t) => t.index === index);
+
+  taskInTasks.completed = check;
+
+  store();
+};
+
+const updateTaskDescription = (index, description) => {
+  const taskInTasks = tasks.find((t) => t.index === index);
+
+  taskInTasks.description = description;
+
+  store();
+};
+
+const inputCreateTask = (description) => {
+  let index = 0;
+
+  if (tasks.length > 0) {
+    index = tasks[tasks.length - 1].index + 1;
+  }
+
+  addTask(description, false, index);
+  store();
+};
+
+export {
```

Comment:
### [Optional]

There should be a function for deleting a task and also for deleting a completed task.

![image](https://user-images.githubusercontent.com/31897434/126054242-c20e4cd4-94c5-4eb6-b3a9-26d15c3dbec8.png)

Sorry for dragging you back, previously missed this for some reason maybe because the app didn't work, that why I tag it as optional but it would be nice to create those functions. 🙏